roles/pulp_content_guard_rbac/README.md

```diff
@@ -0,0 +1,41 @@
+pulp_content_guard_rbac
```

Comment:
The pulp_content_guard role was intended to be generic enough to support other content guards

Reply:
I've moved the RBAC feature under that role by just moving the files and adding an `include_tasks`